src/UI/SlotArea.cpp

```diff
@@ -1114,6 +1116,7 @@ void cSlotAreaAnvil::UpdateResult(cPlayer & a_Player)
 				Output.Empty();
 				SetSlot(2, a_Player, Output);
 				m_ParentWindow.SetProperty(0, 0, a_Player);
+				m_ParentWindow.SendSlot(a_Player, this, 2);
```

Comment:
The output slot doesn't appear to be updating on the client. e.g. if you don't have the SendSlot on line 1204 (sending the output of a valid enchanting), then the client won't show the item even though SetSlot was called.